plugins/invite.js

```diff
@@ -76,6 +76,9 @@ module.exports = {
 
         var addr = server.getAddress()
         var host = ref.parseAddress(addr).host
+        if(typeof host !== 'string') {
+          return cb(new Error('Could not parse host portion from server address:', addr))
```

Comment:
Shouldn't it be `new Error(a + b)` instead of `new Error(a, b)`? I guess, not sure, because the signature of this is `new Error([message[, fileName[, lineNumber]]])`

Reply:
thank you! I was assuming string concat like in `console.log`.